src/confetti.js

```diff
@@ -246,7 +246,12 @@
     ],
     // probably should be true, but back-compat
     disableForReducedMotion: false,
-    scalar: 1
+    scalar: 1,
+    /* New feature 3-D wobble/tilt control  */
+    tiltRange:  [-Math.PI * 0.5,  Math.PI * 0.5], // [min,max] radians
```

Comment:
The original canvas-confetti `tiltAngle` calculation logic:

```javascript
tiltAngle: (Math.random() * (0.75 - 0.25) + 0.25) * Math.PI
// Result: 0.25π ~ 0.75π (45° ~ 135°)
```

The current PR's default value is -90° ~ 90°, but to maintain the original behavior, I think it should be 45° ~ 135°. 

Suggestion: 

```javascript
tiltRange: [Math.PI * 0.25, Math.PI * 0.75] 
```